copy/copy.go

```diff
@@ -383,6 +386,9 @@ func (c *copier) copyMultipleImages(ctx context.Context, policyContext *signatur
 
 	// Determine if we'll need to convert the manifest list to a different format.
 	forceListMIMEType := options.ForceManifestMIMEType
+	if forceListMIMEType == "" && len(options.OverrideManifestMIMETypeList) == 1 {
+		forceListMIMEType = options.OverrideManifestMIMETypeList[0]
```

Comment:
This special case does not make much sense to me — if a single-element array affects `forceListMIMEType`, why does two-element array not do so?

“`OverrideManifestMIMETypeList` overrides the transport-determined list of possible options, but does not change the selection logic otherwise” seems a much simpler story to me.

Reply:
This is one part that I was less clear on. I wanted to make sure that providing a single value in `OverrideManifestMIMETypeList` was equivalent to setting `ForceManifestMIMEType` which required this. I considered a more general approach here. It seems like the more general approach would be that if the list included *only* Docker types, then we'd want to set this to `manifest.DockerV2ListMediaType`, and if the list included *only* OCI types, then we'd set it to `imgspecv1.MediaTypeImageIndex`. I decided not to do this since I realized that if `dest.SupportedManifestMIMETypes` limits to a similar subset, `forceListMIMEType` doesn't get set.

I could just eliminate this, leaving the list conversion alone, although then a single element override list would no longer be equivalent to setting `ForceManifestMIMEType`. Perhaps that's OK, though.